<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Strict//EN">
<html>
<head><title>Sample_5_L001.fastq_20.fastq FastQC Report</title>

<style type="text/css">

 @media screen {
  div.summary {
    width: 18em;
    position:fixed;
    top: 3em;
    margin:1em 0 0 1em;
  }
  
  div.main {
    display:block;
    position:absolute;
    overflow:auto;
    height:auto;
    width:auto;
    top:4.5em;
    bottom:2.3em;
    left:18em;
    right:0;
    border-left: 1px solid #CCC;
    padding:0 0 0 1em;
    background-color: white;
    z-index:1;
  }
  
  div.header {
    background-color: #EEE;
    border:0;
    margin:0;
    padding: 0.5em;
    font-size: 200%;
    font-weight: bold;
    position:fixed;
    width:100%;
    top:0;
    left:0;
    z-index:2;
  }

  div.footer {
    background-color: #EEE;
    border:0;
    margin:0;
	padding:0.5em;
    height: 1.3em;
	overflow:hidden;
    font-size: 100%;
    font-weight: bold;
    position:fixed;
    bottom:0;
    width:100%;
    z-index:2;
  }
  
  img.indented {
    margin-left: 3em;
  }
 }
 
 @media print {
	img {
		max-width:100% !important;
		page-break-inside: avoid;
	}
	h2, h3 {
		page-break-after: avoid;
	}
	div.header {
      background-color: #FFF;
    }
	
 }
 
 body {    
  font-family: sans-serif;   
  color: #000;   
  background-color: #FFF;
  border: 0;
  margin: 0;
  padding: 0;
  }
  
  div.header {
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 200%;
  font-weight: bold;
  width:100%;
  }    
  
  #header_title {
  display:inline-block;
  float:left;
  clear:left;
  }
  #header_filename {
  display:inline-block;
  float:right;
  clear:right;
  font-size: 50%;
  margin-right:2em;
  text-align: right;
  }

  div.header h3 {
  font-size: 50%;
  margin-bottom: 0;
  }
  
  div.summary ul {
  padding-left:0;
  list-style-type:none;
  }
  
  div.summary ul li img {
  margin-bottom:-0.5em;
  margin-top:0.5em;
  }
	  
  div.main {
  background-color: white;
  }
      
  div.module {
  padding-bottom:1.5em;
  padding-top:1.5em;
  }
	  
  div.footer {
  background-color: #EEE;
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 100%;
  font-weight: bold;
  width:100%;
  }


  a {
  color: #000080;
  }

  a:hover {
  color: #800000;
  }
      
  h2 {
  color: #800000;
  padding-bottom: 0;
  margin-bottom: 0;
  clear:left;
  }

  table { 
  margin-left: 3em;
  text-align: center;
  }
  
  th { 
  text-align: center;
  background-color: #000080;
  color: #FFF;
  padding: 0.4em;
  }      
  
  td { 
  font-family: monospace; 
  text-align: left;
  background-color: #EEE;
  color: #000;
  padding: 0.4em;
  }

  img {
  padding-top: 0;
  margin-top: 0;
  border-top: 0;
  }

  
  p {
  padding-top: 0;
  margin-top: 0;
  }
  
</style>

</head>
<body>
<div class="header">
<div id="header_title"><img src="Icons/fastqc_icon.png" alt="FastQC">FastQC Report</div>
<div id="header_filename">
Fri 6 Sep 2013<br />
Sample_5_L001.fastq_20.fastq
</div>
</div>
<div class="summary">
<h2>Summary</h2>
<ul>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M0">Basic Statistics</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M1">Per base sequence quality</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M2">Per sequence quality scores</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M3">Per base sequence content</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M4">Per base GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M5">Per sequence GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M6">Per base N content</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M7">Sequence Length Distribution</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M8">Sequence Duplication Levels</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M9">Overrepresented sequences</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M10">Kmer Content</a></li>
</ul>
</div>
<div class="main">
<div class="module"><h2 id="M0"><img src="Icons/tick.png" alt="[OK]"> Basic Statistics</h2>
<table>
<tr>
<th>Measure</th>
<th>Value</th>
</tr>
<tr>
<td>Filename</td>
<td>Sample_5_L001.fastq_20.fastq</td>
</tr>
<tr>
<td>File type</td>
<td>Conventional base calls</td>
</tr>
<tr>
<td>Encoding</td>
<td>Sanger / Illumina 1.9</td>
</tr>
<tr>
<td>Total Sequences</td>
<td>18689389</td>
</tr>
<tr>
<td>Filtered Sequences</td>
<td>0</td>
</tr>
<tr>
<td>Sequence length</td>
<td>20-51</td>
</tr>
<tr>
<td>%GC</td>
<td>52</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M1"><img src="Icons/tick.png" alt="[OK]"> Per base sequence quality</h2>
<p><img class="indented" src="Images/per_base_quality.png" alt="Per base quality graph"></p>
</div>
<div class="module"><h2 id="M2"><img src="Icons/tick.png" alt="[OK]"> Per sequence quality scores</h2>
<p><img class="indented" src="Images/per_sequence_quality.png" alt="Per Sequence quality graph"></p>
</div>
<div class="module"><h2 id="M3"><img src="Icons/error.png" alt="[FAIL]"> Per base sequence content</h2>
<p><img class="indented" src="Images/per_base_sequence_content.png" alt="Per base sequence content"></p>
</div>
<div class="module"><h2 id="M4"><img src="Icons/error.png" alt="[FAIL]"> Per base GC content</h2>
<p><img class="indented" src="Images/per_base_gc_content.png" alt="Per base GC content graph"></p>
</div>
<div class="module"><h2 id="M5"><img src="Icons/tick.png" alt="[OK]"> Per sequence GC content</h2>
<p><img class="indented" src="Images/per_sequence_gc_content.png" alt="Per sequence GC content graph"></p>
</div>
<div class="module"><h2 id="M6"><img src="Icons/tick.png" alt="[OK]"> Per base N content</h2>
<p><img class="indented" src="Images/per_base_n_content.png" alt="N content graph"></p>
</div>
<div class="module"><h2 id="M7"><img src="Icons/warning.png" alt="[WARN]"> Sequence Length Distribution</h2>
<p><img class="indented" src="Images/sequence_length_distribution.png" alt="Sequence length distribution"></p>
</div>
<div class="module"><h2 id="M8"><img src="Icons/error.png" alt="[FAIL]"> Sequence Duplication Levels</h2>
<p><img class="indented" src="Images/duplication_levels.png" alt="Duplication level graph"></p>
</div>
<div class="module"><h2 id="M9"><img src="Icons/warning.png" alt="[WARN]"> Overrepresented sequences</h2>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Percentage</th>
<th>Possible Source</th>
</tr>
<tr>
<td>GATCGGAAGAGCACACGTCTGAACTCCAGTCACCAGATCATCTCGTATGCC</td>
<td>116768</td>
<td>0.6247823296952083</td>
<td>TruSeq Adapter, Index 7 (100% over 51bp)</td>
</tr>
<tr>
<td>ATATATATATATATATAGATCGGAAGAGCACACGTCTGAACTCCAGTCACC</td>
<td>33750</td>
<td>0.1805837526309715</td>
<td>Illumina Multiplexing PCR Primer 2.01 (100% over 34bp)</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M10"><img src="Icons/warning.png" alt="[WARN]"> Kmer Content</h2>
<p><img class="indented" src="Images/kmer_profiles.png" alt="Kmer graph"></p>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Obs/Exp Overall</th>
<th>Obs/Exp Max</th>
<th>Max Obs/Exp Position</th>
</tr>
<tr>
<td>AAAAA</td>
<td>3279015</td>
<td>5.0325704</td>
<td>8.192125</td>
<td>10</td>
</tr>
<tr>
<td>TTCTT</td>
<td>2587940</td>
<td>3.4742732</td>
<td>6.4361525</td>
<td>6</td>
</tr>
<tr>
<td>TTTTT</td>
<td>2201570</td>
<td>3.4125419</td>
<td>4.686218</td>
<td>11</td>
</tr>
<tr>
<td>CTTCT</td>
<td>2767900</td>
<td>3.2182848</td>
<td>5.4662127</td>
<td>1</td>
</tr>
<tr>
<td>CAGCA</td>
<td>2877370</td>
<td>3.0981467</td>
<td>4.4785094</td>
<td>9</td>
</tr>
<tr>
<td>TCTTC</td>
<td>2546625</td>
<td>2.9610047</td>
<td>5.550598</td>
<td>7</td>
</tr>
<tr>
<td>GAAGA</td>
<td>2201185</td>
<td>2.9317453</td>
<td>11.4145775</td>
<td>6</td>
</tr>
<tr>
<td>CACCA</td>
<td>2733115</td>
<td>2.741427</td>
<td>8.427494</td>
<td>31</td>
</tr>
<tr>
<td>ATATA</td>
<td>1728035</td>
<td>2.662678</td>
<td>9.833654</td>
<td>7</td>
</tr>
<tr>
<td>TCATC</td>
<td>2278235</td>
<td>2.643704</td>
<td>9.171262</td>
<td>38</td>
</tr>
<tr>
<td>TCTTG</td>
<td>1943610</td>
<td>2.4258876</td>
<td>5.137048</td>
<td>7</td>
</tr>
<tr>
<td>ATCAT</td>
<td>1753320</td>
<td>2.344506</td>
<td>9.908731</td>
<td>37</td>
</tr>
<tr>
<td>TATAT</td>
<td>1510515</td>
<td>2.3321202</td>
<td>9.518107</td>
<td>6</td>
</tr>
<tr>
<td>CTCCT</td>
<td>2255235</td>
<td>2.2710683</td>
<td>7.271883</td>
<td>1</td>
</tr>
<tr>
<td>CTCCA</td>
<td>2251875</td>
<td>2.2631996</td>
<td>7.723844</td>
<td>24</td>
</tr>
<tr>
<td>TCCAG</td>
<td>2038120</td>
<td>2.198851</td>
<td>8.051626</td>
<td>25</td>
</tr>
<tr>
<td>AAGAG</td>
<td>1578460</td>
<td>2.1023412</td>
<td>10.041245</td>
<td>7</td>
</tr>
<tr>
<td>TTGAA</td>
<td>1452055</td>
<td>2.0843024</td>
<td>6.0504794</td>
<td>7</td>
</tr>
<tr>
<td>GGAAG</td>
<td>1619325</td>
<td>2.0091634</td>
<td>9.434894</td>
<td>5</td>
</tr>
<tr>
<td>GCCAG</td>
<td>1982640</td>
<td>1.9886653</td>
<td>5.95216</td>
<td>1</td>
</tr>
<tr>
<td>CATCT</td>
<td>1711055</td>
<td>1.9855382</td>
<td>8.39824</td>
<td>39</td>
</tr>
<tr>
<td>GGCAG</td>
<td>1789225</td>
<td>1.9265056</td>
<td>5.218762</td>
<td>1</td>
</tr>
<tr>
<td>ACCAG</td>
<td>1757705</td>
<td>1.8925712</td>
<td>8.100944</td>
<td>32</td>
</tr>
<tr>
<td>GAGCA</td>
<td>1628605</td>
<td>1.8823898</td>
<td>9.151866</td>
<td>9</td>
</tr>
<tr>
<td>AGCAC</td>
<td>1674760</td>
<td>1.803262</td>
<td>8.6870365</td>
<td>10</td>
</tr>
<tr>
<td>CCAGA</td>
<td>1663165</td>
<td>1.7907773</td>
<td>7.854158</td>
<td>33</td>
</tr>
<tr>
<td>AGAGC</td>
<td>1520145</td>
<td>1.7570286</td>
<td>8.830082</td>
<td>8</td>
</tr>
<tr>
<td>CTGAA</td>
<td>1308600</td>
<td>1.626855</td>
<td>8.779</td>
<td>19</td>
</tr>
<tr>
<td>GCACA</td>
<td>1438710</td>
<td>1.5491002</td>
<td>7.8447766</td>
<td>11</td>
</tr>
<tr>
<td>GAACT</td>
<td>1243145</td>
<td>1.5454813</td>
<td>8.507233</td>
<td>21</td>
</tr>
<tr>
<td>TCACC</td>
<td>1506030</td>
<td>1.5136038</td>
<td>7.165365</td>
<td>30</td>
</tr>
<tr>
<td>GTCCA</td>
<td>1395765</td>
<td>1.5058384</td>
<td>5.226079</td>
<td>1</td>
</tr>
<tr>
<td>CCAGT</td>
<td>1393850</td>
<td>1.5037724</td>
<td>7.5401335</td>
<td>26</td>
</tr>
<tr>
<td>TGAAC</td>
<td>1191705</td>
<td>1.481531</td>
<td>8.615705</td>
<td>20</td>
</tr>
<tr>
<td>ATCTC</td>
<td>1274310</td>
<td>1.4787318</td>
<td>7.90579</td>
<td>40</td>
</tr>
<tr>
<td>GATCA</td>
<td>1188010</td>
<td>1.4769374</td>
<td>8.491799</td>
<td>36</td>
</tr>
<tr>
<td>AGTCA</td>
<td>1142060</td>
<td>1.4198122</td>
<td>8.364564</td>
<td>28</td>
</tr>
<tr>
<td>AGATC</td>
<td>1128115</td>
<td>1.4024757</td>
<td>8.332375</td>
<td>35</td>
</tr>
<tr>
<td>CTCGT</td>
<td>1268150</td>
<td>1.3708706</td>
<td>7.3917165</td>
<td>42</td>
</tr>
<tr>
<td>CACAC</td>
<td>1363610</td>
<td>1.367757</td>
<td>7.1420527</td>
<td>12</td>
</tr>
<tr>
<td>TCTGA</td>
<td>1091985</td>
<td>1.360249</td>
<td>8.453076</td>
<td>18</td>
</tr>
<tr>
<td>CAGTC</td>
<td>1188415</td>
<td>1.2821362</td>
<td>7.4289837</td>
<td>27</td>
</tr>
<tr>
<td>TCTCG</td>
<td>1145450</td>
<td>1.238232</td>
<td>7.2337403</td>
<td>41</td>
</tr>
<tr>
<td>CAGAT</td>
<td>958505</td>
<td>1.191616</td>
<td>8.272169</td>
<td>34</td>
</tr>
<tr>
<td>GTCTG</td>
<td>1007995</td>
<td>1.1696925</td>
<td>7.8624783</td>
<td>17</td>
</tr>
<tr>
<td>GTCAC</td>
<td>1040375</td>
<td>1.1224215</td>
<td>7.1958313</td>
<td>29</td>
</tr>
<tr>
<td>AACTC</td>
<td>967290</td>
<td>1.1202402</td>
<td>7.6774607</td>
<td>22</td>
</tr>
<tr>
<td>ATGCC</td>
<td>1019635</td>
<td>1.1000459</td>
<td>7.4701424</td>
<td>47</td>
</tr>
<tr>
<td>CACGT</td>
<td>976385</td>
<td>1.053385</td>
<td>7.138607</td>
<td>14</td>
</tr>
<tr>
<td>ACTCC</td>
<td>1040380</td>
<td>1.0456121</td>
<td>6.755236</td>
<td>23</td>
</tr>
<tr>
<td>TCGTA</td>
<td>765330</td>
<td>0.9533459</td>
<td>7.947861</td>
<td>43</td>
</tr>
<tr>
<td>CGGAA</td>
<td>741160</td>
<td>0.85665447</td>
<td>7.857166</td>
<td>4</td>
</tr>
<tr>
<td>ACACG</td>
<td>791565</td>
<td>0.8523007</td>
<td>6.9740314</td>
<td>13</td>
</tr>
<tr>
<td>CGTCT</td>
<td>777255</td>
<td>0.84021294</td>
<td>7.0957246</td>
<td>16</td>
</tr>
<tr>
<td>GATCG</td>
<td>674390</td>
<td>0.7810246</td>
<td>7.3490973</td>
<td>1</td>
</tr>
<tr>
<td>ACGTC</td>
<td>706540</td>
<td>0.7622594</td>
<td>6.943077</td>
<td>15</td>
</tr>
<tr>
<td>TCGGA</td>
<td>642720</td>
<td>0.74434686</td>
<td>7.8933625</td>
<td>3</td>
</tr>
<tr>
<td>GTATG</td>
<td>541260</td>
<td>0.723761</td>
<td>8.363712</td>
<td>45</td>
</tr>
<tr>
<td>ATCGG</td>
<td>620625</td>
<td>0.71875817</td>
<td>7.361771</td>
<td>2</td>
</tr>
<tr>
<td>TATGC</td>
<td>483920</td>
<td>0.60280293</td>
<td>7.735024</td>
<td>46</td>
</tr>
<tr>
<td>CGTAT</td>
<td>353930</td>
<td>0.44087872</td>
<td>7.5702496</td>
<td>44</td>
</tr>
</table>
</div>
</div><div class="footer">Produced by <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a> (version 0.10.1)</div>
</body></html>