<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Strict//EN">
<html>
<head><title>Sample_2_L001.fastq_30.fastq FastQC Report</title>

<style type="text/css">

 @media screen {
  div.summary {
    width: 18em;
    position:fixed;
    top: 3em;
    margin:1em 0 0 1em;
  }
  
  div.main {
    display:block;
    position:absolute;
    overflow:auto;
    height:auto;
    width:auto;
    top:4.5em;
    bottom:2.3em;
    left:18em;
    right:0;
    border-left: 1px solid #CCC;
    padding:0 0 0 1em;
    background-color: white;
    z-index:1;
  }
  
  div.header {
    background-color: #EEE;
    border:0;
    margin:0;
    padding: 0.5em;
    font-size: 200%;
    font-weight: bold;
    position:fixed;
    width:100%;
    top:0;
    left:0;
    z-index:2;
  }

  div.footer {
    background-color: #EEE;
    border:0;
    margin:0;
	padding:0.5em;
    height: 1.3em;
	overflow:hidden;
    font-size: 100%;
    font-weight: bold;
    position:fixed;
    bottom:0;
    width:100%;
    z-index:2;
  }
  
  img.indented {
    margin-left: 3em;
  }
 }
 
 @media print {
	img {
		max-width:100% !important;
		page-break-inside: avoid;
	}
	h2, h3 {
		page-break-after: avoid;
	}
	div.header {
      background-color: #FFF;
    }
	
 }
 
 body {    
  font-family: sans-serif;   
  color: #000;   
  background-color: #FFF;
  border: 0;
  margin: 0;
  padding: 0;
  }
  
  div.header {
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 200%;
  font-weight: bold;
  width:100%;
  }    
  
  #header_title {
  display:inline-block;
  float:left;
  clear:left;
  }
  #header_filename {
  display:inline-block;
  float:right;
  clear:right;
  font-size: 50%;
  margin-right:2em;
  text-align: right;
  }

  div.header h3 {
  font-size: 50%;
  margin-bottom: 0;
  }
  
  div.summary ul {
  padding-left:0;
  list-style-type:none;
  }
  
  div.summary ul li img {
  margin-bottom:-0.5em;
  margin-top:0.5em;
  }
	  
  div.main {
  background-color: white;
  }
      
  div.module {
  padding-bottom:1.5em;
  padding-top:1.5em;
  }
	  
  div.footer {
  background-color: #EEE;
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 100%;
  font-weight: bold;
  width:100%;
  }


  a {
  color: #000080;
  }

  a:hover {
  color: #800000;
  }
      
  h2 {
  color: #800000;
  padding-bottom: 0;
  margin-bottom: 0;
  clear:left;
  }

  table { 
  margin-left: 3em;
  text-align: center;
  }
  
  th { 
  text-align: center;
  background-color: #000080;
  color: #FFF;
  padding: 0.4em;
  }      
  
  td { 
  font-family: monospace; 
  text-align: left;
  background-color: #EEE;
  color: #000;
  padding: 0.4em;
  }

  img {
  padding-top: 0;
  margin-top: 0;
  border-top: 0;
  }

  
  p {
  padding-top: 0;
  margin-top: 0;
  }
  
</style>

</head>
<body>
<div class="header">
<div id="header_title"><img src="Icons/fastqc_icon.png" alt="FastQC">FastQC Report</div>
<div id="header_filename">
Fri 6 Sep 2013<br />
Sample_2_L001.fastq_30.fastq
</div>
</div>
<div class="summary">
<h2>Summary</h2>
<ul>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M0">Basic Statistics</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M1">Per base sequence quality</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M2">Per sequence quality scores</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M3">Per base sequence content</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M4">Per base GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M5">Per sequence GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M6">Per base N content</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M7">Sequence Length Distribution</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M8">Sequence Duplication Levels</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M9">Overrepresented sequences</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M10">Kmer Content</a></li>
</ul>
</div>
<div class="main">
<div class="module"><h2 id="M0"><img src="Icons/tick.png" alt="[OK]"> Basic Statistics</h2>
<table>
<tr>
<th>Measure</th>
<th>Value</th>
</tr>
<tr>
<td>Filename</td>
<td>Sample_2_L001.fastq_30.fastq</td>
</tr>
<tr>
<td>File type</td>
<td>Conventional base calls</td>
</tr>
<tr>
<td>Encoding</td>
<td>Sanger / Illumina 1.9</td>
</tr>
<tr>
<td>Total Sequences</td>
<td>15984106</td>
</tr>
<tr>
<td>Filtered Sequences</td>
<td>0</td>
</tr>
<tr>
<td>Sequence length</td>
<td>30-51</td>
</tr>
<tr>
<td>%GC</td>
<td>51</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M1"><img src="Icons/tick.png" alt="[OK]"> Per base sequence quality</h2>
<p><img class="indented" src="Images/per_base_quality.png" alt="Per base quality graph"></p>
</div>
<div class="module"><h2 id="M2"><img src="Icons/tick.png" alt="[OK]"> Per sequence quality scores</h2>
<p><img class="indented" src="Images/per_sequence_quality.png" alt="Per Sequence quality graph"></p>
</div>
<div class="module"><h2 id="M3"><img src="Icons/error.png" alt="[FAIL]"> Per base sequence content</h2>
<p><img class="indented" src="Images/per_base_sequence_content.png" alt="Per base sequence content"></p>
</div>
<div class="module"><h2 id="M4"><img src="Icons/error.png" alt="[FAIL]"> Per base GC content</h2>
<p><img class="indented" src="Images/per_base_gc_content.png" alt="Per base GC content graph"></p>
</div>
<div class="module"><h2 id="M5"><img src="Icons/tick.png" alt="[OK]"> Per sequence GC content</h2>
<p><img class="indented" src="Images/per_sequence_gc_content.png" alt="Per sequence GC content graph"></p>
</div>
<div class="module"><h2 id="M6"><img src="Icons/tick.png" alt="[OK]"> Per base N content</h2>
<p><img class="indented" src="Images/per_base_n_content.png" alt="N content graph"></p>
</div>
<div class="module"><h2 id="M7"><img src="Icons/warning.png" alt="[WARN]"> Sequence Length Distribution</h2>
<p><img class="indented" src="Images/sequence_length_distribution.png" alt="Sequence length distribution"></p>
</div>
<div class="module"><h2 id="M8"><img src="Icons/error.png" alt="[FAIL]"> Sequence Duplication Levels</h2>
<p><img class="indented" src="Images/duplication_levels.png" alt="Duplication level graph"></p>
</div>
<div class="module"><h2 id="M9"><img src="Icons/warning.png" alt="[WARN]"> Overrepresented sequences</h2>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Percentage</th>
<th>Possible Source</th>
</tr>
<tr>
<td>GATCGGAAGAGCACACGTCTGAACTCCAGTCACTGACCAATCTCGTATGCC</td>
<td>45638</td>
<td>0.2855211295520688</td>
<td>TruSeq Adapter, Index 4 (100% over 51bp)</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M10"><img src="Icons/warning.png" alt="[WARN]"> Kmer Content</h2>
<p><img class="indented" src="Images/kmer_profiles.png" alt="Kmer graph"></p>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Obs/Exp Overall</th>
<th>Obs/Exp Max</th>
<th>Max Obs/Exp Position</th>
</tr>
<tr>
<td>AAAAA</td>
<td>2935485</td>
<td>4.807526</td>
<td>7.678847</td>
<td>10</td>
</tr>
<tr>
<td>TTCTT</td>
<td>2415490</td>
<td>3.3536024</td>
<td>6.6192365</td>
<td>6</td>
</tr>
<tr>
<td>CTTCT</td>
<td>2553970</td>
<td>3.2047813</td>
<td>7.2482886</td>
<td>1</td>
</tr>
<tr>
<td>CAGCA</td>
<td>2409085</td>
<td>3.1088688</td>
<td>4.2821107</td>
<td>9</td>
</tr>
<tr>
<td>TTTTT</td>
<td>1991850</td>
<td>3.0597556</td>
<td>4.3966064</td>
<td>5</td>
</tr>
<tr>
<td>TCTTC</td>
<td>2311585</td>
<td>2.900631</td>
<td>5.3835063</td>
<td>7</td>
</tr>
<tr>
<td>GAAGA</td>
<td>1688300</td>
<td>2.7080262</td>
<td>7.4798737</td>
<td>6</td>
</tr>
<tr>
<td>TCTTG</td>
<td>1716155</td>
<td>2.3883939</td>
<td>5.1716127</td>
<td>7</td>
</tr>
<tr>
<td>CTTCA</td>
<td>1853825</td>
<td>2.356208</td>
<td>5.7754507</td>
<td>1</td>
</tr>
<tr>
<td>CTTGG</td>
<td>1670210</td>
<td>2.330041</td>
<td>5.7388678</td>
<td>1</td>
</tr>
<tr>
<td>CTCCT</td>
<td>2006715</td>
<td>2.2758543</td>
<td>9.718228</td>
<td>1</td>
</tr>
<tr>
<td>CTGCT</td>
<td>1729400</td>
<td>2.17531</td>
<td>5.4429708</td>
<td>1</td>
</tr>
<tr>
<td>CTCCA</td>
<td>1850935</td>
<td>2.1262398</td>
<td>9.802438</td>
<td>1</td>
</tr>
<tr>
<td>TCCTC</td>
<td>1827530</td>
<td>2.072637</td>
<td>6.7472625</td>
<td>2</td>
</tr>
<tr>
<td>TTGAA</td>
<td>1291975</td>
<td>2.0410419</td>
<td>5.8752847</td>
<td>7</td>
</tr>
<tr>
<td>TCCAG</td>
<td>1587020</td>
<td>2.0219498</td>
<td>6.545095</td>
<td>2</td>
</tr>
<tr>
<td>TCCTT</td>
<td>1527110</td>
<td>1.9162533</td>
<td>5.6436987</td>
<td>2</td>
</tr>
<tr>
<td>AAGAG</td>
<td>1179080</td>
<td>1.8912394</td>
<td>6.025235</td>
<td>7</td>
</tr>
<tr>
<td>CCTCC</td>
<td>1800095</td>
<td>1.8451444</td>
<td>5.2496934</td>
<td>1</td>
</tr>
<tr>
<td>GGAAG</td>
<td>1160170</td>
<td>1.8416437</td>
<td>6.0217905</td>
<td>5</td>
</tr>
<tr>
<td>GAGCA</td>
<td>1209005</td>
<td>1.7303948</td>
<td>5.709788</td>
<td>9</td>
</tr>
<tr>
<td>CCCAG</td>
<td>1449555</td>
<td>1.6691635</td>
<td>5.701598</td>
<td>1</td>
</tr>
<tr>
<td>AGCAC</td>
<td>1263430</td>
<td>1.6304274</td>
<td>5.453026</td>
<td>10</td>
</tr>
<tr>
<td>AGAGC</td>
<td>1097135</td>
<td>1.5702803</td>
<td>5.2767925</td>
<td>8</td>
</tr>
<tr>
<td>GTCCA</td>
<td>1166015</td>
<td>1.4855664</td>
<td>6.205732</td>
<td>1</td>
</tr>
</table>
</div>
</div><div class="footer">Produced by <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a> (version 0.10.1)</div>
</body></html>