<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Strict//EN">
<html>
<head><title>Sample_5_L001.fastq_30.fastq FastQC Report</title>

<style type="text/css">

 @media screen {
  div.summary {
    width: 18em;
    position:fixed;
    top: 3em;
    margin:1em 0 0 1em;
  }
  
  div.main {
    display:block;
    position:absolute;
    overflow:auto;
    height:auto;
    width:auto;
    top:4.5em;
    bottom:2.3em;
    left:18em;
    right:0;
    border-left: 1px solid #CCC;
    padding:0 0 0 1em;
    background-color: white;
    z-index:1;
  }
  
  div.header {
    background-color: #EEE;
    border:0;
    margin:0;
    padding: 0.5em;
    font-size: 200%;
    font-weight: bold;
    position:fixed;
    width:100%;
    top:0;
    left:0;
    z-index:2;
  }

  div.footer {
    background-color: #EEE;
    border:0;
    margin:0;
	padding:0.5em;
    height: 1.3em;
	overflow:hidden;
    font-size: 100%;
    font-weight: bold;
    position:fixed;
    bottom:0;
    width:100%;
    z-index:2;
  }
  
  img.indented {
    margin-left: 3em;
  }
 }
 
 @media print {
	img {
		max-width:100% !important;
		page-break-inside: avoid;
	}
	h2, h3 {
		page-break-after: avoid;
	}
	div.header {
      background-color: #FFF;
    }
	
 }
 
 body {    
  font-family: sans-serif;   
  color: #000;   
  background-color: #FFF;
  border: 0;
  margin: 0;
  padding: 0;
  }
  
  div.header {
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 200%;
  font-weight: bold;
  width:100%;
  }    
  
  #header_title {
  display:inline-block;
  float:left;
  clear:left;
  }
  #header_filename {
  display:inline-block;
  float:right;
  clear:right;
  font-size: 50%;
  margin-right:2em;
  text-align: right;
  }

  div.header h3 {
  font-size: 50%;
  margin-bottom: 0;
  }
  
  div.summary ul {
  padding-left:0;
  list-style-type:none;
  }
  
  div.summary ul li img {
  margin-bottom:-0.5em;
  margin-top:0.5em;
  }
	  
  div.main {
  background-color: white;
  }
      
  div.module {
  padding-bottom:1.5em;
  padding-top:1.5em;
  }
	  
  div.footer {
  background-color: #EEE;
  border:0;
  margin:0;
  padding: 0.5em;
  font-size: 100%;
  font-weight: bold;
  width:100%;
  }


  a {
  color: #000080;
  }

  a:hover {
  color: #800000;
  }
      
  h2 {
  color: #800000;
  padding-bottom: 0;
  margin-bottom: 0;
  clear:left;
  }

  table { 
  margin-left: 3em;
  text-align: center;
  }
  
  th { 
  text-align: center;
  background-color: #000080;
  color: #FFF;
  padding: 0.4em;
  }      
  
  td { 
  font-family: monospace; 
  text-align: left;
  background-color: #EEE;
  color: #000;
  padding: 0.4em;
  }

  img {
  padding-top: 0;
  margin-top: 0;
  border-top: 0;
  }

  
  p {
  padding-top: 0;
  margin-top: 0;
  }
  
</style>

</head>
<body>
<div class="header">
<div id="header_title"><img src="Icons/fastqc_icon.png" alt="FastQC">FastQC Report</div>
<div id="header_filename">
Fri 6 Sep 2013<br />
Sample_5_L001.fastq_30.fastq
</div>
</div>
<div class="summary">
<h2>Summary</h2>
<ul>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M0">Basic Statistics</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M1">Per base sequence quality</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M2">Per sequence quality scores</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M3">Per base sequence content</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M4">Per base GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M5">Per sequence GC content</a></li>
<li><img src="Icons/tick.png" alt="[PASS]"> <a href="#M6">Per base N content</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M7">Sequence Length Distribution</a></li>
<li><img src="Icons/error.png" alt="[FAIL]"> <a href="#M8">Sequence Duplication Levels</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M9">Overrepresented sequences</a></li>
<li><img src="Icons/warning.png" alt="[WARNING]"> <a href="#M10">Kmer Content</a></li>
</ul>
</div>
<div class="main">
<div class="module"><h2 id="M0"><img src="Icons/tick.png" alt="[OK]"> Basic Statistics</h2>
<table>
<tr>
<th>Measure</th>
<th>Value</th>
</tr>
<tr>
<td>Filename</td>
<td>Sample_5_L001.fastq_30.fastq</td>
</tr>
<tr>
<td>File type</td>
<td>Conventional base calls</td>
</tr>
<tr>
<td>Encoding</td>
<td>Sanger / Illumina 1.9</td>
</tr>
<tr>
<td>Total Sequences</td>
<td>18214157</td>
</tr>
<tr>
<td>Filtered Sequences</td>
<td>0</td>
</tr>
<tr>
<td>Sequence length</td>
<td>30-51</td>
</tr>
<tr>
<td>%GC</td>
<td>52</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M1"><img src="Icons/tick.png" alt="[OK]"> Per base sequence quality</h2>
<p><img class="indented" src="Images/per_base_quality.png" alt="Per base quality graph"></p>
</div>
<div class="module"><h2 id="M2"><img src="Icons/tick.png" alt="[OK]"> Per sequence quality scores</h2>
<p><img class="indented" src="Images/per_sequence_quality.png" alt="Per Sequence quality graph"></p>
</div>
<div class="module"><h2 id="M3"><img src="Icons/error.png" alt="[FAIL]"> Per base sequence content</h2>
<p><img class="indented" src="Images/per_base_sequence_content.png" alt="Per base sequence content"></p>
</div>
<div class="module"><h2 id="M4"><img src="Icons/error.png" alt="[FAIL]"> Per base GC content</h2>
<p><img class="indented" src="Images/per_base_gc_content.png" alt="Per base GC content graph"></p>
</div>
<div class="module"><h2 id="M5"><img src="Icons/tick.png" alt="[OK]"> Per sequence GC content</h2>
<p><img class="indented" src="Images/per_sequence_gc_content.png" alt="Per sequence GC content graph"></p>
</div>
<div class="module"><h2 id="M6"><img src="Icons/tick.png" alt="[OK]"> Per base N content</h2>
<p><img class="indented" src="Images/per_base_n_content.png" alt="N content graph"></p>
</div>
<div class="module"><h2 id="M7"><img src="Icons/warning.png" alt="[WARN]"> Sequence Length Distribution</h2>
<p><img class="indented" src="Images/sequence_length_distribution.png" alt="Sequence length distribution"></p>
</div>
<div class="module"><h2 id="M8"><img src="Icons/error.png" alt="[FAIL]"> Sequence Duplication Levels</h2>
<p><img class="indented" src="Images/duplication_levels.png" alt="Duplication level graph"></p>
</div>
<div class="module"><h2 id="M9"><img src="Icons/warning.png" alt="[WARN]"> Overrepresented sequences</h2>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Percentage</th>
<th>Possible Source</th>
</tr>
<tr>
<td>GATCGGAAGAGCACACGTCTGAACTCCAGTCACCAGATCATCTCGTATGCC</td>
<td>111885</td>
<td>0.6142749291114598</td>
<td>TruSeq Adapter, Index 7 (100% over 51bp)</td>
</tr>
<tr>
<td>ATATATATATATATATAGATCGGAAGAGCACACGTCTGAACTCCAGTCACC</td>
<td>31383</td>
<td>0.17230004111636898</td>
<td>Illumina Multiplexing PCR Primer 2.01 (100% over 34bp)</td>
</tr>
</table>
</div>
<div class="module"><h2 id="M10"><img src="Icons/warning.png" alt="[WARN]"> Kmer Content</h2>
<p><img class="indented" src="Images/kmer_profiles.png" alt="Kmer graph"></p>
<table>
<tr>
<th>Sequence</th>
<th>Count</th>
<th>Obs/Exp Overall</th>
<th>Obs/Exp Max</th>
<th>Max Obs/Exp Position</th>
</tr>
<tr>
<td>AAAAA</td>
<td>3134125</td>
<td>4.8522434</td>
<td>7.942362</td>
<td>10</td>
</tr>
<tr>
<td>TTCTT</td>
<td>2520185</td>
<td>3.44429</td>
<td>6.2510715</td>
<td>6</td>
</tr>
<tr>
<td>TTTTT</td>
<td>2077855</td>
<td>3.2563415</td>
<td>4.386902</td>
<td>11</td>
</tr>
<tr>
<td>CTTCT</td>
<td>2693705</td>
<td>3.2104795</td>
<td>5.4739685</td>
<td>1</td>
</tr>
<tr>
<td>CAGCA</td>
<td>2797635</td>
<td>3.110464</td>
<td>4.4625907</td>
<td>9</td>
</tr>
<tr>
<td>TCTTC</td>
<td>2485395</td>
<td>2.9622064</td>
<td>5.502548</td>
<td>7</td>
</tr>
<tr>
<td>GAAGA</td>
<td>2137945</td>
<td>2.9227893</td>
<td>11.522085</td>
<td>6</td>
</tr>
<tr>
<td>CACCA</td>
<td>2661835</td>
<td>2.7532032</td>
<td>8.528888</td>
<td>31</td>
</tr>
<tr>
<td>TCATC</td>
<td>2234335</td>
<td>2.6565042</td>
<td>9.232114</td>
<td>38</td>
</tr>
<tr>
<td>ATATA</td>
<td>1616605</td>
<td>2.515044</td>
<td>9.063079</td>
<td>7</td>
</tr>
<tr>
<td>TCTTG</td>
<td>1889080</td>
<td>2.420178</td>
<td>5.0480633</td>
<td>7</td>
</tr>
<tr>
<td>ATCAT</td>
<td>1714985</td>
<td>2.3324494</td>
<td>9.875224</td>
<td>37</td>
</tr>
<tr>
<td>CTCCT</td>
<td>2189530</td>
<td>2.2757437</td>
<td>7.2059245</td>
<td>1</td>
</tr>
<tr>
<td>CTCCA</td>
<td>2188405</td>
<td>2.2690418</td>
<td>7.811863</td>
<td>24</td>
</tr>
<tr>
<td>TCCAG</td>
<td>1975210</td>
<td>2.2014313</td>
<td>8.131698</td>
<td>25</td>
</tr>
<tr>
<td>TATAT</td>
<td>1410620</td>
<td>2.1999323</td>
<td>8.728261</td>
<td>6</td>
</tr>
<tr>
<td>AAGAG</td>
<td>1536825</td>
<td>2.1009967</td>
<td>10.096137</td>
<td>7</td>
</tr>
<tr>
<td>TTGAA</td>
<td>1418765</td>
<td>2.0741465</td>
<td>6.096738</td>
<td>7</td>
</tr>
<tr>
<td>GGAAG</td>
<td>1571110</td>
<td>2.018343</td>
<td>9.594318</td>
<td>5</td>
</tr>
<tr>
<td>GCCAG</td>
<td>1917995</td>
<td>2.0038674</td>
<td>6.042498</td>
<td>1</td>
</tr>
<tr>
<td>CATCT</td>
<td>1670305</td>
<td>1.9859028</td>
<td>8.41227</td>
<td>39</td>
</tr>
<tr>
<td>GGCAG</td>
<td>1723385</td>
<td>1.9354454</td>
<td>5.261041</td>
<td>1</td>
</tr>
<tr>
<td>ACCAG</td>
<td>1708220</td>
<td>1.899232</td>
<td>8.257303</td>
<td>32</td>
</tr>
<tr>
<td>GAGCA</td>
<td>1584660</td>
<td>1.8938577</td>
<td>9.248987</td>
<td>9</td>
</tr>
<tr>
<td>AGCAC</td>
<td>1626645</td>
<td>1.808535</td>
<td>8.736873</td>
<td>10</td>
</tr>
<tr>
<td>CCAGA</td>
<td>1620405</td>
<td>1.8015976</td>
<td>7.9497304</td>
<td>33</td>
</tr>
<tr>
<td>AGAGC</td>
<td>1477975</td>
<td>1.7663565</td>
<td>8.916645</td>
<td>8</td>
</tr>
<tr>
<td>CTGAA</td>
<td>1269430</td>
<td>1.6184168</td>
<td>8.769424</td>
<td>19</td>
</tr>
<tr>
<td>GCACA</td>
<td>1394630</td>
<td>1.5505763</td>
<td>7.9182773</td>
<td>11</td>
</tr>
<tr>
<td>GAACT</td>
<td>1201525</td>
<td>1.5318434</td>
<td>8.482402</td>
<td>21</td>
</tr>
<tr>
<td>TCACC</td>
<td>1464935</td>
<td>1.5189137</td>
<td>7.2460136</td>
<td>30</td>
</tr>
<tr>
<td>CCAGT</td>
<td>1355160</td>
<td>1.5103668</td>
<td>7.620769</td>
<td>26</td>
</tr>
<tr>
<td>GTCCA</td>
<td>1352040</td>
<td>1.5068895</td>
<td>5.1748176</td>
<td>1</td>
</tr>
<tr>
<td>ATCTC</td>
<td>1246885</td>
<td>1.482479</td>
<td>7.964689</td>
<td>40</td>
</tr>
<tr>
<td>GATCA</td>
<td>1158540</td>
<td>1.4770412</td>
<td>8.528283</td>
<td>36</td>
</tr>
<tr>
<td>TGAAC</td>
<td>1156970</td>
<td>1.4750396</td>
<td>8.647181</td>
<td>20</td>
</tr>
<tr>
<td>AGTCA</td>
<td>1117640</td>
<td>1.4248972</td>
<td>8.466231</td>
<td>28</td>
</tr>
<tr>
<td>AGATC</td>
<td>1093985</td>
<td>1.394739</td>
<td>8.394539</td>
<td>35</td>
</tr>
<tr>
<td>CTCGT</td>
<td>1229870</td>
<td>1.3740697</td>
<td>7.500937</td>
<td>42</td>
</tr>
<tr>
<td>CACAC</td>
<td>1326355</td>
<td>1.3718824</td>
<td>7.2149057</td>
<td>12</td>
</tr>
<tr>
<td>TCTGA</td>
<td>1061185</td>
<td>1.3562207</td>
<td>8.485771</td>
<td>18</td>
</tr>
<tr>
<td>CAGTC</td>
<td>1153540</td>
<td>1.2856551</td>
<td>7.5648932</td>
<td>27</td>
</tr>
<tr>
<td>TCTCG</td>
<td>1117045</td>
<td>1.2480164</td>
<td>7.3517137</td>
<td>41</td>
</tr>
<tr>
<td>CAGAT</td>
<td>939010</td>
<td>1.1971588</td>
<td>8.303184</td>
<td>34</td>
</tr>
<tr>
<td>GTCTG</td>
<td>984335</td>
<td>1.1821417</td>
<td>7.9709272</td>
<td>17</td>
</tr>
<tr>
<td>GTCAC</td>
<td>1009330</td>
<td>1.1249287</td>
<td>7.283663</td>
<td>29</td>
</tr>
<tr>
<td>AACTC</td>
<td>937010</td>
<td>1.1113446</td>
<td>7.66979</td>
<td>22</td>
</tr>
<tr>
<td>ATGCC</td>
<td>989510</td>
<td>1.1028389</td>
<td>7.636637</td>
<td>47</td>
</tr>
<tr>
<td>ACTCC</td>
<td>1008110</td>
<td>1.045256</td>
<td>6.8132973</td>
<td>23</td>
</tr>
<tr>
<td>CACGT</td>
<td>937550</td>
<td>1.0449278</td>
<td>7.2218513</td>
<td>14</td>
</tr>
<tr>
<td>TCGTA</td>
<td>743295</td>
<td>0.94994956</td>
<td>8.043911</td>
<td>43</td>
</tr>
<tr>
<td>CGGAA</td>
<td>710730</td>
<td>0.84940714</td>
<td>7.9037476</td>
<td>4</td>
</tr>
<tr>
<td>ACACG</td>
<td>761905</td>
<td>0.84710056</td>
<td>7.0379434</td>
<td>13</td>
</tr>
<tr>
<td>CGTCT</td>
<td>751655</td>
<td>0.839785</td>
<td>7.138806</td>
<td>16</td>
</tr>
<tr>
<td>GATCG</td>
<td>650750</td>
<td>0.77962023</td>
<td>7.4203386</td>
<td>1</td>
</tr>
<tr>
<td>ACGTC</td>
<td>677675</td>
<td>0.75528926</td>
<td>7.0077596</td>
<td>15</td>
</tr>
<tr>
<td>TCGGA</td>
<td>618450</td>
<td>0.7409238</td>
<td>7.9453797</td>
<td>3</td>
</tr>
<tr>
<td>GTATG</td>
<td>531330</td>
<td>0.7299291</td>
<td>8.4753685</td>
<td>45</td>
</tr>
<tr>
<td>ATCGG</td>
<td>600220</td>
<td>0.7190836</td>
<td>7.4376245</td>
<td>2</td>
</tr>
<tr>
<td>TATGC</td>
<td>475285</td>
<td>0.607426</td>
<td>7.857907</td>
<td>46</td>
</tr>
<tr>
<td>CGTAT</td>
<td>345100</td>
<td>0.44104633</td>
<td>7.6591096</td>
<td>44</td>
</tr>
</table>
</div>
</div><div class="footer">Produced by <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a> (version 0.10.1)</div>
</body></html>